roles/jenkins_builder/tasks/pkgs.yml

```diff
@@ -64,6 +64,9 @@
   - python-argparse
 # used for utils/create-files.py
   - pyxattr
+# used for cli
+  - ncurses-devel
+  - readline
```

Comment:
I think you mean "readline-devel" ? (and I found that out after I merged and pushed :/)